Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -12051,6 +12052,62 @@ def get_server_config():
     server_config = body.get('sysConf', {})
     return server_config
 
+
+# Type variable to indicate that the decorated function can be any callable
```

Comment:
won't just using `Callable` work? 🤔

Reply:
Techincally yes, but doing so this way will preserve a few important things, like the original typing of the function it wraps, and is more type safe.